CHANGELOG.md

```diff
@@ -31,6 +31,11 @@ The types of changes are:
 - Load default privacy notices on startup [#3401](https://github.com/ethyca/fides/pull/3401/files)
 - Add privacy centre button text customisations [#3432](https://github.com/ethyca/fides/pull/3432)
 - Add privacy centre favicon customisation [#3432](https://github.com/ethyca/fides/pull/3432)
+- Add ability for `fides-js` to make API calls to Fides [#3361](https://github.com/ethyca/fides/pull/3361)
```

Comment:
do we want to consolidate all the changelog items as part of this feat branch once we merge into `main`?

Reply:
I was under the impression we only document the PRs merged directly into `main` and not PRs as part of a feat branch, but either way works for me

Reply:
oh oops I didn't see your second comment until just now after I've merged 🤦‍♀️ let's ask the team!